python/neuroglancer/lib.py

```diff
@@ -419,19 +452,21 @@ def expand_to_chunk_size(self, chunk_size, offset=Vec(0,0,0, dtype=int)):
     result.maxpt = np.ceil(result.maxpt / chunk_size) * chunk_size 
     return result + offset
 
-  def shrink_to_chunk_size(self, chunk_size):
+  def shrink_to_chunk_size(self, chunk_size, offset=Vec(0,0,0, dtype=int)):
```

Comment:
You're right. Documentation is a good idea. You've got the right idea, the offset is the dataset voxel offset.